packages/video-ad-sdk/src/adUnit/helpers/dom/elementObservers/onElementResize.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable promise/prefer-await-to-callbacks */
-import debounce from 'lodash/debounce';
+import debounce from 'lodash.debounce';
```

Comment:
if we start adding more modules of lodash, then importing the normal lodash is better if we use lodash-webpack-plugin

Reply:
I don't really plan on adding more to be honest.